vt100_output_windows.go

```diff
@@ -0,0 +1,299 @@
+// +build windows
```

Comment:
I thought about windows console doesn't understand vt100 escape sequences. but It's not true?
If it is true, I think this file should be named `windows_output.go` and should rename `VT100Writer`.

I'll check this later.